### DISCUSSION TITLE
Using the mortals sheet for an Orpheus game - how can i change the name of some advantages?

### DISCUSSION THREAD

**Discussion by jehsse** [JohanFalt/Foundry_WoD20#891] [Q&A] [answered]
I'm using the mortals sheet for an Orpheus chronicle - but it needs some adaptations. Since I cannot create new advantages or labels, is there a way I could change the name of some of the existent advantages - and other fields like "derangement", or "roleplaying tips"?

**Comment by JohanFalt** [comment]
Easiest way would be to alter the language file.

**Comment by JohanFalt** [comment]
See here: https://github.com/JohanFalt/Foundry_WoD20/issues/907

**Comment by JohanFalt** [comment, marked as answer]
Will arrive with v4.2
